spec/lib/importers/revision_score_importer_spec.rb

```diff
@@ -189,6 +193,8 @@
           # revision 12345. But it works so far.
           expect(wiki.revisions.first.wp10).to be_between(0, 100)
         end
+
+        expect(wikidata.revisions.first.features).not_to be_empty
```

Comment:
The `fetch_ores_data_for_revision_id` method is only used for Wikipedia so far, happens within a controller action, and is not part of the `update_revision_scores_for_all_wikis` data flow, but you're right that it would be good to add a test to make sure the fetch-by-id method works for Wikidata if/when we ever want to use it for that.